utils/tweet_query.py

```diff
@@ -97,6 +96,10 @@ def pull(self, query:str, output_dir: str,
 				print('\n' + '-'*30)
 				break
 
+			# Avoiding 1 request/sec rate limit
+			end_time_req = time.time()
+			time.sleep(1.0 - (end_time_req-start_time_req))
```

Comment:
This gives negative numbers if the query takes more than 1 second. The sleep time should be precomputed as the max of 0 and 1 - dt.